lib/tidewave/router.ex

```diff
@@ -219,13 +219,14 @@ defmodule Tidewave.Router do
     |> halt()
   end
 
-  defp tidewave_html() do
+  defp tidewave_html(plug_config) do
     client_url = Application.get_env(:tidewave, :client_url, "https://tidewave.ai")
 
     config = %{
       project_name: MCP.project_name(),
       framework_type: "phoenix",
-      tidewave_version: package_version(:tidewave)
+      tidewave_version: package_version(:tidewave),
+      team: plug_config[:team] && Map.new(plug_config[:team])
```

Comment:
Would this be better?

```suggestion
      team: Map.new(plug_config[:team] || %{)
```

Any way is fine by me :)